src/openrct2/world/TileElement.h

```diff
@@ -193,22 +195,18 @@ enum
     MAP_ELEM_TRACK_SEQUENCE_GREEN_LIGHT = (1 << 7),
 };
 
-enum
-{
-    MAP_ELEM_SMALL_SCENERY_COLOUR_FLAG_NEEDS_SUPPORTS = (1 << 5),
-};
 
-#define TILE_ELEMENT_QUADRANT_MASK 0xC0
-#define TILE_ELEMENT_TYPE_MASK 0x3C
-#define TILE_ELEMENT_DIRECTION_MASK 0x03
+
+#define TILE_ELEMENT_QUADRANT_MASK 0xC0 // bits 6 + 7
+#define TILE_ELEMENT_TYPE_MASK 0x3C     // bits 2 + 3 + 4 + 5
+#define TILE_ELEMENT_DIRECTION_MASK 0x03 // bits 0 + 1
```

Comment:
Why not use a binary literal instead?

Reply:
That would be fine as well. I do think that if we go down that route, we should put it in the code style rules and adopt this for the whole codebase, for consistency reasons.